<launch>

    <!-- parametre -->
    <arg name="robot_namespace" default="gen3_lite" />

    <!-- launch turtbo script -->
    <node name='kinova_action_client' pkg='kinova' type='action_client.py' ns="$(arg robot_namespace)" output="screen" />
</launch>
